core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -479,7 +479,7 @@ class CivilizationInfo {
 
     fun addNotification(text: String, color: Color, action: NotificationAction?=null) {
         if (playerType == PlayerType.AI) return // no point in lengthening the saved game info if no one will read it
-        notifications.add(Notification(text, color, action))
+        notifications.add(Notification(text.tr(), color, action))
```

Comment:
This is not a good solution, since this means that notifications are 'frozen' in the language of the adder - in multiplayer games, each person could have a different language, so we need to save the notifications untranslated

Reply:
Hadn't that on radar, thanks for the info. I'll check for a sufficient solution to fix the issue without impact to multiplayer and update the PR. Maybe it's enough to revert this change and just keep the mod part.

Reply:
I reverted it.

Calling tr() in this method wasn't necessary, because the NotificationsScroll class that shows the notifications does the required work. It calls for every notification text the [toLabel()](https://github.com/yairm210/Unciv/blob/master/core/src/com/unciv/ui/worldscreen/NotificationsScroll.kt#L33) method, which triggers the translation for this client. Due the other change, the active mods are taken into account and the correct translation is used.

So again learned something about Unciv code. :smile: